### DISCUSSION TITLE
DHCP Server not pulling config data

### DISCUSSION THREAD

**Discussion by yoshideryu** [RaspAP/raspap-webgui#943] [Q&A] [answered]
i am not sure if i missed something but the DHCP Server page dose not pull any of the current data in always shows me static interface as dhcp

**Comment by billz** [comment]
Can you be more specific? ie, what DHCP settings are you seeing versus expected?

**Reply by yoshideryu** [reply]
wlan1 is set and working for a static ip  but the page show DHCP and shows blank ip and 0.0.0.0 for the mask. this interface is also the DHCP server the DHCP options are picking up the correct info  

so i am guessing the issue is something with reading the dhcpcd.conf ?

**Comment by yoshideryu** [comment, marked as answer]
I was able to correct this issue by deleting the interface config in the dhcpcd.conf and resaving from the WEBGUI looking at the format from RaspAP  it is very different from what was here before i must have edited this in the past when i was having issues and it did not like the way i formatted it
